src/utils/debug.ts

```diff
@@ -1,5 +1,8 @@
 export const debug = {
   log: (...args: any[]) => {
+    if (process.env.NODE_ENV === "production") {
```

Comment:
I did a little research to see how this might work across bundlers, because our current build with `tsc` will leave this intact, I think. it seems that with vite and webpack, this will be transformed as needed, but I would prefer that the bundler replace this with `true` for the build we ship just to avoid any issues at build or runtime

Reply:
![image](https://github.com/acao/codemirror-json-schema/assets/1368727/f14f1cb3-16ec-486f-9584-03ef816db23b)

it looks like `tsc` just strips out the statement entirely! this at least prevents bundling or runtime issues, but it means our shipped bundle will be noisily debugging. I wonder if we can configure tsconfig in some way to do the above

Reply:
ok, I've added a quick `sed` script to address this, though I don't think it will be an issue, as most frameworks take care of this, just want to be sure for edge cases!

Reply:
![image](https://github.com/acao/codemirror-json-schema/assets/1368727/6cf8df4c-3f30-4a6f-8147-00e0f66bbc7e)

Reply:
eventually we could use vite library mode or even something simple like `tsup` to take care of this and provide a umd export/etc, but this should do for now!